polaris-terraform/pipeline-terraform/dev.tfvars

```diff
@@ -31,11 +31,11 @@ overnight_clear_down = {
 }
 
 sliding_clear_down = {
-  disabled       = 0
-  look_back_days = 1
-  protect_blobs  = false
-  schedule       = "0 */5 * * * *"
-  batch_size     = 5
+  disabled        = 0
```

Comment:
Do QA and DEV so that they clear down cases over 12 hours old so that every day's activity is cleared the following night.